src/main/java/ch/njol/skript/conditions/CondEntityUnload.java

```diff
@@ -0,0 +1,33 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.entity.LivingEntity;
+
+@Name("Can Despawn")
+@Description("Check if an entity can despawn when the chunk they're located at is unloaded.")
```

Comment:
Maybe a note should be put here, regarding how MC decides what/when entities despawn.
Or maybe redirect to the wiki: https://minecraft.wiki/w/Mob_spawning#Despawning